app/tests/test_keycloak_manager.py

```diff
@@ -80,3 +78,24 @@ def test_multiple_transferring_body_user_groups_returns_corresponding_bodies(
                 "foo",
                 "bar",
             ]
+
+    @patch(
+        "app.main.authorize.keycloak_manager.keycloak.KeycloakOpenID.introspect"
+    )
+    def test_transferring_body_user_groups_one_invalid_valid_group(self, app):
+        """
+        Given a list of keycloak groups including 2 transferring bodies
+        When calling get_user_transferring_body_keycloak_groups with it
```

Comment:
this docstring doesnt seem to be matching whats happening in the actual test logic

Reply:
updated docstring to reflect the correct details of unit test